nginx/server/sites.sls

```diff
@@ -75,6 +75,8 @@ nginx_init_{{ site.host.name }}_tls:
 
 {% endif %}
 
+{%- if grains.get('virtual_subtype', None) not in ['Docker', 'LXC'] %}
```

Comment:
Why you don't want to create sites configuration on Docker?
Wrap only what doesn't work on these platforms.